code/modules/modular_computers/computers/item/role_tablet_presets.dm

```diff
@@ -159,7 +159,7 @@
 	greyscale_colors = "#D99A2E#69DBF3#E3DF3D"
 	starting_programs = list(
 		/datum/computer_file/program/supermatter_monitor,
-		/datum/computer_file/program/crew_manifest, // SKYRAT EDIT ADDITION - Manifests for all crew
+
```

Comment:
however you will want to remove all the whitespace

Reply:
though this did present a problem with existing crew manifests on security and heads.. hmm